advanced/am-pattern-2/README.md

```diff
@@ -9,10 +9,33 @@
 
```

Comment:
@Chathuru Please recheck the README instructions. I saw missing sub headings in the case of instructions for running this chart from the Helm Hub (e.g. see [sample](https://github.com/wso2/kubernetes-is/tree/master/advanced/is-pattern-1#install-chart-from-wso2-helm-chart-repository)).

Also, please update the Helm chart version to map with the new release.

Reply:
Update the readme to the correct helm chart version. What are the missing sub headings

Reply:
@Chathuru the one I found is `Install Chart From WSO2 Helm Chart Repository`. Please recheck the whole doc again.